src/components/pages/dashboard-pages/HomeStudent.jsx

```diff
@@ -0,0 +1,57 @@
+import { Link } from 'react-router-dom';
+
+const HomeStudent = ({ profile, onNavigate, error }) => {
+  const { firstName, lastName, dateOfBirth, adultName } = profile;
+  let studentAge = (new Date().getTime() - new Date(dateOfBirth)) / (24 * 3600 * 365.25 * 1000);
```

Comment:
This function is ok here for now. But if we find our components having a lot of utility functions we can move them to their own module and import them.